tutorials/01_install.md

```diff
@@ -2,5 +2,75 @@
 
 Next Tutorial: \ref commandline
 
-Refer to [Ignition GUI's entry page](https://ignitionrobotics.org/libs/gui)
-for install instructions.
+[Install](#install)
+
+* [Binary Install](#binary-install)
+
+* [Source Install](#source-install)
+
+    * [Prerequisites](#prerequisites)
+
+    * [Building from Source](#building-from-source)
+
+# Install
+
+We recommend following the [Binary Install](#binary-install) instructions to get up and running as quickly and painlessly as possible.
+
+The [Source Install](#source-install) instructions should be used if you need the very latest software improvements, if you need to modify the code, or if you plan to make a contribution.
+
+## Binary Install
+
+On Ubuntu Bionic, it's possible to install Ignition GUI's version 1 as follows:
+
+Add OSRF packages:
+
+    echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list
+    echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-prerelease `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-prerelease.list
+    sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
+    sudo apt update
+
+Install Ignition GUI:
+
+    sudo apt install libignition-gui4-dev
```

Comment:
Let's make this about a generic `ign-gui` version `N`. This one hasn't been released yet.